tests/records/test_extract.py

```diff
@@ -35,7 +35,7 @@ def test_init():
         'test_link',
         1,
         1.0,
-        'test_legend'
+        {'lang': 'test_legend'}
```

Comment:
It will not fail with `lang` as lang ?

Reply:
No, it's not getting checked here. But I agree, it is confusing. I change it.